app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -466,6 +466,8 @@ def center_toolbar_filename_classic
       elsif to_display_center.include?(@display)
         return "#{@display}_center"
       end
+    elsif @display == 'generic_objects'
+      return "#{@display}_center"
```

Comment:
why the interpolation when we know the value?

do we want to prevent people grepping for `generic_objects_center`?